deploy/charts/vault-secrets-webhook/README.md.gotmpl

```diff
@@ -0,0 +1,190 @@
+{{- define "chart.kubeVersionBadge" -}}
+{{- if .KubeVersion -}}![kube version: {{ .KubeVersion }}](https://img.shields.io/badge/kube%20version-{{ .KubeVersion | replace "-" "--" }}-informational?style=flat-square){{- end -}}
```

Comment:
I'd like it better if the text said `kubernetes version:`, also as I mentioned in a previous comment, it is currently not added to the final Readme as it looks for a `kubeVersion` field in the Chart.yaml, which is currently missing. Is this badge necessary btw?

Reply:
Removed, not needed